app/src/rgb_underglow.c

```diff
@@ -335,7 +340,8 @@ int zmk_rgb_underglow_toggle() {
 }
 
 int zmk_rgb_underglow_set_hsb(struct zmk_led_hsb color) {
-    if (color.h > HUE_MAX || color.s > SAT_MAX || color.b > BRT_MAX) {
+    if (color.h > HUE_MAX || color.s > SAT_MAX || color.b < CONFIG_ZMK_RGB_UNDERGLOW_BRT_MIN ||
+        color.b > CONFIG_ZMK_RGB_UNDERGLOW_BRT_MAX) {
```

Comment:
[This should still be addressed](https://github.com/zmkfirmware/zmk/pull/669#discussion_r584978501). A keymap can definitely have a key that sets brightness to 100%, and that should be 100% (or 0% conversely) of the `CONFIG_ZMK_RGB_UNDERGLOW_BRT_MAX`, not an error. A user of a keymap shouldn't have to know the hardware limitations and wonder why their key to set brightness to 100% (max!) doesn't work.

This could also open up the possible discussion of using 0-100% in the state always, but when actually updating the LEDs, we use the % of the hardware like `(max-min) * state brightness + min`